code/datums/status_effects/debuffs.dm

```diff
@@ -947,11 +949,11 @@
 
 /datum/status_effect/slimegrub
 	id = "grub_infection"
-	duration = 60 SECONDS //a redgrub infestation in a slime 
```

Comment:
have no idea why these are marked with a change

Reply:
removing a trailing space. it's part of your auto-formatter. it's fine.